airflow/config_templates/default_airflow.cfg

```diff
@@ -917,23 +917,17 @@ max_tis_per_query = 512
 # scheduler at once
 use_row_level_locking = True
 
-# Max number of DAGs to create DagRuns for per scheduler loop
-#
-# Default: 10
-# max_dagruns_to_create_per_loop =
+# Max number of DAGs to create DagRuns for per scheduler loop.
+max_dagruns_to_create_per_loop = 10
```

Comment:
Should this be "10" since it's a string?

Reply:
it doesn't matter if it's quoted or not and I don't see any string options quoted in this file (which is automatically rendered from docs anyways).